src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -465,15 +465,19 @@ private void check_remote_url(final String repositoryName) throws InterruptedExc
         assertTrue("remote URL has not been updated", remotes.contains(localMirror()));
     }
 
+    private Collection<String> getBranchNames(Set<Branch> branches) {
```

Comment:
The description in the commit message does not fit the subject and changes in the code (anymore).